.idea/vcs.xml

```diff
@@ -2,5 +2,6 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/oppia-android" vcs="Git" />
   </component>
-</project>
+</project>
```

Comment:
Please revert this file changes. We do not want these unnecessary changes.

Reply:
I will do this. I am sorry for the confusion

Reply:
No issues. To make PRs more effective, before sending the PR for review its nice to go to the pull request, click on "Files Changed" and try to self review it first and whatever mistakes you find, fix them and then send it for review. This will help you a lot in becoming a good developer.

Reply:
Same here. This file changes are also not needed.